### DISCUSSION TITLE
Unified Alerting - Data sources selected, but alert rules from all sources are constantly re-loaded

### DISCUSSION THREAD

**Discussion by mac133k** [grafana/grafana#72913] [Alerting]
**Current behaviour:**
This is a very useful feature to allow selecting one or more datasources in Alerting > Alert rules (https://github.com/grafana/grafana/issues/62969), but it seems like even if only one or more datasources are selected and the view is filtered to only display the rules from those, all datasources are being checked and a message is displayed saying 'Loading rules from <number> datasources".

**Expected behavriour:**
Only rules from selected datasource(s) are being re-loaded in Alert rules GUI.

Grafana version: 10.0.1

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82542. Feel free to continue the discussion around this there. Thank you!
